plugins/services/src/js/constants/VolumeProfile.js

```diff
@@ -0,0 +1,5 @@
+var VolumeProfile = {
```

Comment:
Shouldn't we put all possible values for `profile` in this constants file?

Reply:
Great catch, I actually had the wrong naming here. @nLight showed me the [RAML definition in marathon](https://github.com/mesosphere/marathon/blob/master/docs/docs/rest-api/public/api/v2/types/volumes.raml#L48) and it should be profileName. So it's not really an enum of defined profiles, but a general name you can set.